marketplace-service/src/main/java/com/axonivy/market/constants/GitHubConstants.java

```diff
@@ -8,7 +8,7 @@ public class GitHubConstants {
   public static final String AXONIVY_MARKET_ORGANIZATION_NAME = "axonivy-market";
   public static final String AXONIVY_MARKETPLACE_REPO_NAME = "market";
   public static final String AXONIVY_MARKETPLACE_PATH = "market";
-  public static final String DEFAULT_BRANCH = "feature/MARP-463-Multilingualism-for-Website";
+  public static final String DEFAULT_BRANCH = "master";
```

Comment:
can we make it as a runtime variable like market.cors.allowed.origin.patterns=${MARKET_CORS_ALLOWED_ORIGIN}
then we can modify without commit new code